<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qscapeDialogBase</class>
 <widget class="QDialog" name="qscapeDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>912</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>738</width>
    <height>912</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QScape</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>572</width>
         <height>696</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <property name="minimumSize">
         <size>
          <width>708</width>
          <height>794</height>
         </size>
        </property>
        <attribute name="title">
         <string>General</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_16">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="inRastLabel">
               <property name="text">
                <string>Land cover classification:</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QComboBox" name="inRastCombo">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="editable">
                  <bool>false</bool>
                 </property>
                 <property name="currentText">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="massLabel">
               <property name="text">
                <string>Classes containing mass objects (1, 2, ...) [optional]:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="massLineEdit">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Classes containing mass objects&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Insert class IDs of mass objects e.g. built-up and forest (delimited by commas). &lt;/p&gt;&lt;p&gt;This prevents viewshed calculation on top of these preventing contamination.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="inDEMLabel">
               <property name="text">
                <string>Height model:</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QComboBox" name="inDEMCombo">
                 <property name="mouseTracking">
                  <bool>false</bool>
                 </property>
                 <property name="editable">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_13">
             <item>
              <widget class="QLabel" name="heightLabel">
               <property name="text">
                <string>Viewing elevation above the ground (m) [optional]:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="heightDSpinBox">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Viewing elevation above the ground&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Height of the observer's eyes.&lt;/p&gt;&lt;p&gt;Standard value: 1.75&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="whatsThis">
                <string/>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
               <property name="value">
                <double>1.750000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_14">
             <item>
              <widget class="QLabel" name="radiusLabel">
               <property name="text">
                <string>Maximum visibility radius (m) [optional]:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="radiusDSpinBox">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Maximum visibility radius&lt;/span&gt;&lt;/p&gt;&lt;p&gt;How far the observer is able to recognise elements in the landscape.&lt;/p&gt;&lt;p&gt;Minimum value: -1 (infinity)&lt;br/&gt;Standard value: 1400&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>-1.000000000000000</double>
               </property>
               <property name="maximum">
                <double>10000.000000000000000</double>
               </property>
               <property name="value">
                <double>1400.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="envGroupBox">
             <property name="title">
              <string>Environment:</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_8">
                  <item>
                   <widget class="QRadioButton" name="globalRadioBut">
                    <property name="toolTip">
                     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Global&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Model the entire study area.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                    <property name="text">
                     <string>Global</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QGroupBox" name="globalGroupBox">
                    <property name="title">
                     <string/>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_11">
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_6">
                       <item>
                        <widget class="QLabel" name="globalLabel">
                         <property name="text">
                          <string>Lag (m) [optional]:</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QSpinBox" name="globalSpinBox">
                         <property name="toolTip">
                          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Lag&lt;/span&gt;&lt;/p&gt;&lt;p&gt;The amount of meters between land cover pixels to be sampled.&lt;/p&gt;&lt;p&gt;Higher lag for better performance.&lt;br/&gt;Lower lag for better representation (enables potential mapping back into space).&lt;/p&gt;&lt;p&gt;Minimum value: 0&lt;br/&gt;Standard value: 0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                         </property>
                         <property name="toolTipDuration">
                          <number>-1</number>
                         </property>
                         <property name="suffix">
                          <string/>
                         </property>
                         <property name="minimum">
                          <number>0</number>
                         </property>
                         <property name="maximum">
                          <number>100000</number>
                         </property>
                         <property name="value">
                          <number>0</number>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_10">
                  <item>
                   <widget class="QRadioButton" name="singleRadioBut">
                    <property name="enabled">
                     <bool>true</bool>
                    </property>
                    <property name="toolTip">
                     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Single point&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Study a single observer point in the landscape.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                    <property name="text">
                     <string>Single point</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QGroupBox" name="singleGroupBox">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="title">
                     <string/>
                    </property>
                    <property name="checkable">
                     <bool>false</bool>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_12">
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_9">
                       <item>
                        <widget class="QLabel" name="singleLabel">
                         <property name="text">
                          <string>Coordinate viewing position (x, y):</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_3">
                         <item>
                          <widget class="QLineEdit" name="singleLineEdit">
                           <property name="text">
                            <string>0.0,0.0</string>
                           </property>
                           <property name="clearButtonEnabled">
                            <bool>false</bool>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QToolButton" name="singleToolBut">
                           <property name="text">
                            <string>...</string>
                           </property>
                           <property name="checkable">
                            <bool>false</bool>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="metricsGroupBox">
             <property name="title">
              <string>Spatial metrics:</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QGroupBox" name="openGroupBox">
                  <property name="title">
                   <string>Openness:</string>
                  </property>
                  <layout class="QGridLayout" name="gridLayout">
                   <item row="0" column="0">
                    <layout class="QHBoxLayout" name="horizontalLayout_7">
                     <item>
                      <widget class="QGroupBox" name="sizeGroupBox">
                       <property name="title">
                        <string>Size:</string>
                       </property>
                       <layout class="QGridLayout" name="gridLayout_4">
                        <item row="0" column="0">
                         <layout class="QVBoxLayout" name="verticalLayout_19">
                          <item>
                           <widget class="QRadioButton" name="areaRadioBut">
                            <property name="text">
                             <string>Area</string>
                            </property>
                            <property name="checked">
                             <bool>true</bool>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QRadioButton" name="varRadioBut">
                            <property name="text">
                             <string>Visible Area
Ratio</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QRadioButton" name="coreRadioBut">
                            <property name="text">
                             <string>Core Area</string>
                            </property>
                            <property name="checked">
                             <bool>false</bool>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </item>
                       </layout>
                      </widget>
                     </item>
                     <item>
                      <widget class="QGroupBox" name="shapeGroupBox">
                       <property name="title">
                        <string>Shape:</string>
                       </property>
                       <layout class="QGridLayout" name="gridLayout_5">
                        <item row="0" column="0">
                         <layout class="QVBoxLayout" name="verticalLayout_18">
                          <item>
                           <widget class="QRadioButton" name="ratioRadioBut">
                            <property name="text">
                             <string>Perimeter-Area 
Ratio</string>
                            </property>
                            <property name="checked">
                             <bool>true</bool>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QRadioButton" name="shapeRadioBut">
                            <property name="text">
                             <string>Shape Index</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QRadioButton" name="fractalRadioBut">
                            <property name="text">
                             <string>Fractal Dimension 
Index</string>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </item>
                       </layout>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <widget class="QGroupBox" name="heteroGroupBox">
                    <property name="title">
                     <string>Heterogeneity:</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_2">
                     <item row="0" column="0">
                      <layout class="QVBoxLayout" name="verticalLayout_17">
                       <item>
                        <widget class="QRadioButton" name="absRadioBut">
                         <property name="layoutDirection">
                          <enum>Qt::LeftToRight</enum>
                         </property>
                         <property name="text">
                          <string>Absolute number 
of land cover types</string>
                         </property>
                         <property name="checked">
                          <bool>true</bool>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QRadioButton" name="simpsonRadioBut">
                         <property name="text">
                          <string>Simpson's Diversity</string>
                         </property>
                         <property name="checked">
                          <bool>false</bool>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QRadioButton" name="shannonRadioBut">
                         <property name="text">
                          <string>Shannon's Entropy</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QGroupBox" name="complexGroupBox">
                    <property name="title">
                     <string>Complexity:</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_3">
                     <item row="0" column="0">
                      <layout class="QVBoxLayout" name="verticalLayout_20">
                       <item>
                        <widget class="QRadioButton" name="altRadioBut">
                         <property name="text">
                          <string>Alternation of 
land cover</string>
                         </property>
                         <property name="checked">
                          <bool>true</bool>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QRadioButton" name="contagionRadioBut">
                         <property name="text">
                          <string>Contagion Index</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_15">
             <item>
              <widget class="QLabel" name="memoryLabel">
               <property name="text">
                <string>Amount of memory to use in MB [optional]:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="memorySpinBox">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Amount of memory to use in MB&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Minimum value: 1&lt;br/&gt;Standard value: 500&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>64000</number>
               </property>
               <property name="singleStep">
                <number>1</number>
               </property>
               <property name="value">
                <number>500</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QLabel" name="outLabel">
               <property name="text">
                <string>Output:</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="_2">
               <item>
                <widget class="QLineEdit" name="outLineEdit"/>
               </item>
               <item>
                <widget class="QToolButton" name="outToolBut">
                 <property name="text">
                  <string>Browse</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <property name="minimumSize">
         <size>
          <width>708</width>
          <height>794</height>
         </size>
        </property>
        <attribute name="title">
         <string>About</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_11"/>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QGroupBox" name="statusGroupBox">
         <property name="title">
          <string>Status:</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="statusLabel">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>Complete the input fields and press OK...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
         <property name="centerButtons">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>qscapeDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>769</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>qscapeDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>769</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
